src/main/resources/org/synyx/urlaubsverwaltung/core/mail/sicknote_end_of_sick_pay.ftl

```diff
@@ -1,5 +1,4 @@
-Hallo ${sickNote.person.niceName},
```

Comment:
In case of office user it is not clear which person is ment by this notification. Or is there any mention in mail subject?
Maybe we can add the `${sickNote.person.niceName}` somewhere below?